src/routes/styleguide/__layout.reset.svelte

```diff
@@ -0,0 +1,25 @@
+<script context="module" lang="ts">
+  import { mode } from '$app/env';
+
+  /** @type {import('@sveltejs/kit').Load} */
+  export async function load({}) {
+    let responseObj: any = {
```

Comment:
A tiny nit that you can ignore: Why even store this in a variable? You _could_ just return the error if they're not in development and then just go about your business.

```js
import type { LoadInput } from '@sveltejs/kit';

export async function load({ fetch }: LoadInput) {
   if (mode !== 'development') return { status: 404, error: 'Not Found' };

  return { status: 200 };
}
```